CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.4.0
+## 2.4.1
```

Comment:
Should there be a new entry for 2.4.1?

Reply:
Yes to match pubspec. I published 2.4.0 prematurely, retracted it shortly after